qiskit/algorithms/gradients/lin_comb_estimator_gradient.py

```diff
@@ -44,23 +50,35 @@ class LinCombEstimatorGradient(BaseEstimatorGradient):
     `arXiv:1811.11184 <https://arxiv.org/pdf/1811.11184.pdf>`_
     """
 
-    def __init__(self, estimator: BaseEstimator, **options):
-        """
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        derivative_type: DerivativeType = DerivativeType.REAL,
+        **options,
+    ):
+        r"""
         Args:
             estimator: The estimator used to compute the gradients.
+            derivative_type: The type of derivative. Can be either ``DerivativeType.REAL``
+                ``DerivativeType.IMAG``, or ``DerivativeType.COMPLEX``. Defaults to
+                ``DerivativeType.REAL``.
+                * ``DerivativeType.REAL`` computes :math:`2 \mathrm{Re}[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉]`.
```

Comment:
This still all wraps as paragraph text 

![image](https://user-images.githubusercontent.com/40241007/192801305-14073f29-f015-4aef-9405-5dde095fd88a.png)


I recall us in the past trying to find ways to have lists come out. Not sure adding a blank line helps, at least this is my vague recollection from the past which is why we ended up with something more complex in Nature where we did a list of items https://github.com/Qiskit/qiskit-nature/blob/d0ac5d9cfeb65f5db37a9ea2972309cd84190f29/qiskit_nature/circuit/library/ansatzes/ucc.py#L142

At this point close to release I think the resultant docs are ok - it could be improved upon but I think we should get this merged as its ok and perhaps look at improving it if we want later.

Reply:
hmm. I'm not sure why it's not working. I added a blank line and used `+` instead of `*`. If the changed one doesn't work, I'm stuck.

Reply:
As I mentioned above we struggled in the past - you can see from the link what we ended up with - this is how it comes out https://qiskit.org/documentation/nature/stubs/qiskit_nature.circuit.library.UCC.html see excitations parameter (even then there are bulleted sub-lists I see which are wrapping)

Reply:
Oh, `+` is literally displayed as `+`. I thought it's some secret command. Let me try `-`, which looks working nicely e.g. 
- qubit_converter
- num_particles
- num_spin_orbitals

Reply:
You can try - that is in the main text not under parameters. It was making lists under a parameter description we struggled with - ended up using :xxxxx: as you see for items.

Reply:
I did not think how it looked in the image was too bad - I know its not what you wanted but something like this can always be looked at more later with a view to improving the resultant format.

Reply:
This seems the best so far and I would leave it like this. The first sentence, instead of following straight on from the parameter is split to the next line, but the list is a bullet list of items as you wanted

![image](https://user-images.githubusercontent.com/40241007/193093161-a6d1cee1-bba8-48c4-8d1b-0c409274a48b.png)

Reply:
Cool!